packages/grafbase-sdk/tests/unit/graphql.test.ts

```diff
@@ -65,11 +67,11 @@ describe('GraphQL connector', () => {
   })
 
   it('combines multiple apis into one extension', () => {
-    const contentful = connector.GraphQL({
+    const contentful = connector.GraphQL('Contentful', {
       url: 'https://graphql.contentful.com/content/v1/spaces/{{ env.CONTENTFUL_SPACE_ID }}/environments/{{ env.CONTENTFUL_ENVIRONMENT }}'
     })
 
-    const github = connector.GraphQL({
+    const github = connector.GraphQL('GitHug', {
```

Comment:
```suggestion
    const github = connector.GraphQL('GitHub', {
```

Intentional?

Reply:
Intentional. I want the string to be different from the namespace here.